api/service/legacysync/syncing.go

```diff
@@ -807,6 +858,12 @@ func (ss *StateSync) UpdateBlockAndStatus(block *types.Block, bc *core.BlockChai
 		Str("blockHex", block.Hash().Hex()).
 		Uint32("ShardID", block.ShardID()).
 		Msg("[SYNC] UpdateBlockAndStatus: New Block Added to Blockchain")
+
+	if verifyAllSig && haveCurrentSig {
+		if err := bc.WriteCommitSig(block.NumberU64(), block.GetCurrentCommitSig()); err != nil {
```

Comment:
no need to explicitly write to DB here. You can just put the commigSigbitmap inside the Block object and the insertChain() will commit the the sig as long as it's present inside the block.

Reply:
I feel a little unconfident for writing signature to db that is not verified. Do you think writing unverified commit sig might cause some problem?

Reply:
After discussion, it's safe to say this will not be a problem. This field in DB is only used by leader proposing a new block when bootstrap. So if the leader node has a wrong commit sig in DB, it will only effect itself and a view change will be expected later. Removed `bc.WriteCommitSig` in both downloader and DNS sync logic.